scraper/src/strategies/default_strategy.py

```diff
@@ -190,7 +190,7 @@ def _get_text_content_for_level(self, node, current_level, selectors):
         if 'attributes' in selectors[current_level]:
             attributes = {}
             for attribute_name in list(selectors[current_level][
-                'attributes'].keys()):
+                                           'attributes'].keys()):
```

Comment:
unrelated to this commit, please undo

Reply:
It is a fix of a wrong indentation, would rather keep it

Reply:
Not in this commit, it's unrelated. The commit speaks about using format() instead of explicit calls to str(); it is not about fixing indentation here and there.

If you want to re-layout the code, do it in a dedicated commit - and setup an automated check on the coding style.